### DISCUSSION TITLE
chat history

### DISCUSSION THREAD

**Discussion by MatFia00** [BearWare/TeamTalk5#2030] [Q&A]
Hello,

Is it possible to save chat history? This concerns the situation when one of the users turns off the application and re-enters it, or will be able to see what was written in the chat. If such a service is available, how to enable it?

**Comment by lbk2907** [comment]
For now is not possible.

**Comment by bear101** [comment]
Storing user conversations server side causes problems with [GPDR](https://gdpr-info.eu/). For now you can only store conversations locally on your device.
